src/modules/fetcher/FetcherModule.js

```diff
@@ -3,33 +3,37 @@
 import { NativeAppEventEmitter } from 'react-native'
 import NativeFetcherModule from './NativeFetcherModule'
 
+export type TargetFilePathsType = {[path: string]: string}
+
+export type FailureMessageType = { [url: string]: string }
+
 export type FetchResultType = {
-  failureMessages: { [url: string]: string },
-  successFilePaths: { [url: string]: string }
+  failureMessages: FailureMessageType,
+  fetchedUrls: { [path: string]: { lastUpdate: string, url: string} }
```

Comment:
Missing exact type:
`
   fetchedUrls: { [path: string]: {| lastUpdate: string, url: string |} } 
`

Reply:
Done in some other PR ;)